### DISCUSSION TITLE
Conflicts with Decky

### DISCUSSION THREAD

**Discussion by gatlinnewhouse** [CrankshaftForSteam/crankshaft#16] [Bug Report]
https://github.com/SteamDeckHomebrew/decky-loader/issues/143

any way that crankshaft could be made compatible with steamdeckhomebrew? maybe just host on a different port?

**Comment by coolavery** [comment]
Hi, I don't think this is an issue with the ports conflicting, I got both Decky and Crankshaft working on my desktop yesterday without having to modify ports. The fix for Decky compatibility is pretty small, it should be in the next Crankshaft release.

For reference, the only two ports that Crankshaft uses are Steam's debugging port (which afaik doesn't have any problems handling multiple connections), and a port where it hosts the plugin API (by default 8085). These are configurable with command line flags:

```bash
./crankshaft -h
Usage of ./crankshaft:
  [...]
  -debug-port string
        CEF debug port (default "8080")
  [...]
  -server-port string
        Port to run HTTP/websocket server on (default "8085")
  [...]
```

**Comment by gatlinnewhouse** [comment]
@coolavery thanks for the quick reply, I will try out changing the server-port to see if I can get decky working, otherwise I will just update when the next release occurs!

**Comment by coolavery** [comment]
Created a ticket on Sourcehut: https://todo.sr.ht/~avery/crankshaft/68

We'll be closing the Github issues on this repo and consolidating into Sourcehut tickets, converting this to a discussion instead!
